### DISCUSSION TITLE
关于产品的几个想法

### DISCUSSION THREAD

**Discussion by anno189** [usememos/memos#301] [Ideas]
至少现阶段看来，产品对标的浮墨。
暂时还没有看到作者的定位。

假设是浮墨，那么我把这个产品定位在闪念笔记的层面上。

以下是我关于闪念笔记的想法。
我把每个memos 都理解成闪念笔记，或者说卡片。
那么我期望在增加如下功能：
1. 标题，有标题才有思考，才有记录。
2. 时间编辑，期望能编辑创建时间，能到年月日就好了。我会记录或者回溯一些内容，这些内容是有时间的。如果今天记录的是昨天发生的内容，会觉得怪怪的。或者我会提前记录一个东西，不是任务。这点和任务管理不一样，或许就是提醒吧。
3. 编辑器，能标记字体的颜色，删除线，这类简单的 markdown 语法。不需要表格什么复杂的东西。毕竟是卡片，不是 markdown 编辑器。能做到提醒就好了。
4. memos 显示，能标记出来，info/warning/error/ok等等，类似 ob中的 `> [!warning] `

关于卡片整理的逻辑还没有想好。
以上仅供参考。

作为一款开源产品，还是很感谢作者的贡献。最后期望能尽快看到作者对于产品的定位和思考。

另：0.4.5手机打开不能上传文件，0.6.0还没有试。

**Comment by Zeng1998** [comment]
按作者的想法是尽可能保持编辑器简单，所以第三点应该不太可能，而且你说的这两点，颜色和删除线，都不是markdown语法

**Comment by lmm214** [comment]
第2点是有的，点时间可以更改

**Reply by anno189** [reply]
收到，这个功能我非常需要，可以考虑一些迁移的事情了。
